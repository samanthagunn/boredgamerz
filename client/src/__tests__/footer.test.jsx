import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "../components/footer";
import renderer from "react-test-renderer";

describe("The footer component...", () => {
  test("renders without crashing", () => {
    const { baseElement } = render(<Footer />);
    expect(baseElement).toBeDefined();
  });
  test("matches the snapshot.", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
