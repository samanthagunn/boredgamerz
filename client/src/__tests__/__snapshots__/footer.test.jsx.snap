// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The footer component... matches the snapshot. 1`] = `
<ion-footer>
  <ion-toolbar
    color="tertiary"
  >
    <div>
      <p>
        Â© 2020 Bored Gamerz, BravoBunch
      </p>
    </div>
  </ion-toolbar>
</ion-footer>
`;
